test/test_required_base_class.py

```diff
@@ -0,0 +1,69 @@
+"""Unit tests for required-base-classes."""
+
+import astroid
+from pylint.testutils import CheckerTestCase, Message, set_config
+
+from edx_lint.pylint.required_base_class import RequiredBaseClassChecker
+
+
+class RequiredBaseClassTestCase(CheckerTestCase):
+    """Unittest tests of RequiredBaseClassChecker."""
+
+    CHECKER_CLASS = RequiredBaseClassChecker
+
+    def get_class_node(self, code):
+        """Parse `code`, and return the last class node.
+
+        The code should have at least one class definition.
+
+        """
+        node = astroid.parse(code)
```

Comment:
Maybe you can use astroid.testutils.extract_node / astroid.extract_node for this? It looks something as in:

```
from astroid import extract_node
node = extract_node('''
class A:
    pass
class B: #@
   pass
class C:
   pass
''')
```